src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -12,6 +12,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
 <Project ToolsVersion="14.0">
 
+  <PropertyGroup>
+    <!--
+      Defines the list of file extensions that VS will monitor to reload the application.
+      We'll only define these for projects targeting RazorLangVersion 3.0 or later, and let VS use defaults in other cases.
+    -->
+    <UpToDateReloadFileTypes Condition="'$(_Targeting30OrNewerRazorLangVersion)' == 'true'">$(UpToDateReloadFileTypes);.cs;.razor;.resx;.cshtml</UpToDateReloadFileTypes>
```

Comment:
Since we're specifying `.cs` we should be limiting this to C# projects to not impact other languages that for whatever reason have our bits on 😆